examples/net_server/main.rs

```diff
@@ -34,12 +33,12 @@ fn main() -> Result<()> {
 pub struct State1;
 impl SimpleState for State1 {
     fn on_start(&mut self, data: StateData<'_, GameData<'_, '_>>) {
-        data.world
-            .create_entity()
-            .with(NetConnection::<String>::new(
-                "127.0.0.1:3457".parse().unwrap(),
-            ))
-            .build();
+        //        data.world
```

Comment:
reason why this is commented out?

Reply:
Yes, not needed anymore, since connections are created dynamically, let me remove that.